src/Bicep.LangServer.UnitTests/Handlers/BicepDecompileForPasteCommandHandlerTests.cs

```diff
@@ -1488,11 +1491,18 @@ public async Task JsonValue_Invalid_CantConvert(string json)
         [DataRow(
             "[1]",
             PasteType.BicepValue, // Valid json and valid Bicep expression
-            @"[1]")]
+            @"[
```

Comment:
Is this intentional?  Doesn't seem right.

Reply:
It is, because the decompiler always inserts a newline before the first item of the synthetic array syntax node.

Reply:
Btw this won't be a breaking change for users because it matches the behavior of the v1 formatter.